spectral_cube/tests/test_stokes_spectral_cube.py

```diff
@@ -46,8 +46,8 @@ def test_direct_init_invalid_shape(self, use_dask):
         assert exc.value.args[0] == "All spectral cubes should have the same shape"
 
     @pytest.mark.parametrize('component', ('I', 'Q', 'U', 'V', 'RR', 'LL', 'RL', 'LR', 'XX', 'XY', 'YX', 'YY',
-                                           'RX', 'RY', 'LX', 'LY', 'XR,', 'XL', 'YR', 'YL', 'PP', 'PQ', 'QP', 'QQ', 
-                                           'RCircular', 'LCircular', 'Linear', 'Ptotal', 'Plinear', 'PFtotal', 
+                                           'RX', 'RY', 'LX', 'LY', 'XR', 'XL', 'YR', 'YL', 'PP', 'PQ', 'QP', 'QQ',
+                                           'RCircular', 'LCircular', 'Linear', 'Ptotal', 'Plinear', 'PFtotal',
                                            'PFlinear', 'Pangle'))
```

Comment:
again a minor future thing: we could import this list of stuff to loop over from `stokes_spectral_cube`.  But, maybe the redundancy is good here so that the test is checking something hard-coded.

Reply:
Did as you suggested/